UI/Selenium/Steps/ConsultationRoomSteps.cs

```diff
@@ -110,5 +110,75 @@ public void ThenAllParticipantsLeaveConsultationRoom()
                 Driver.FindElement(ParticipantWaitingRoomPage.JoinPrivateMeetingButton).Displayed.Should().BeTrue();
             }
         }
+
+        [Then(@"both participants click on the leave button to leave the room")]
+        public void ThenBothParticipantsLeaveConsultationRoom()
+        {
+            var participants = _hearing.Participant.Where(p => !p.Id.ToLower().Contains("judge")).ToList();
```

Comment:
I had another conversation about this.  It's important to use the **role** of the participant for the correct type not the id, username or email. 

For example "Mr Justice A Smyth" could be a judge user, or a winger, JOH or panel member.  The role tells you which is correct.

Reply:
you mean Party? Party has clearly defined names "Claimant, Defendant, Observer, Panel member". I think Observer and Panel member are treated like staff. And there is no party/role defined for Judge. We have to depend on Id. Is my understanding correct?

Reply:
@vsdevelope, I had changes requested earlier, can be achieved by 

var judge = _hearing.Participant.Where(a => a.Role.Name.ToLower().Contains("judge")).FirstOrDefault();

Reply:
Used "party" to identify non-staff participants